assets/src/edit-story/app/story/useStoryReducer/reducers/setBackgroundElement.js

```diff
@@ -77,7 +89,16 @@ function setBackgroundElement( state, { elementId } ) {
 			pageElements,
 			elementPosition,
 			0,
-		);
+		).map( ( element ) => {
+			// Set isBackground for the element.
+			if ( element.id === elementId ) {
+				return {
+					...element,
+					isBackground: true,
+				};
+			}
+			return element;
```

Comment:
There's only one nuance here: do we remove "background" marker anywhere? If so, somewhere we need to either set `isBackground: false` or delete `isBackground` key.

Reply:
Background is [removed above](https://github.com/google/web-stories-wp/pull/194/files#diff-67540d27f97b3e21fff3dc40b61011f8R56) only when setting background element to `null` (unsetting background without adding new).

Note that when setting a new background and the previous existed, the previous background element will be completely removed -- there is no need to set the previous element's `isBackground` to `false` since the element is removed anyway.